files/usr/share/cinnamon/applets/menu@cinnamon.org/applet.js

```diff
@@ -3037,27 +3037,25 @@ class CinnamonMenuApplet extends Applet.TextIconApplet {
     _listApplications(pattern){
         let res = [];
         let exactMatch = null;
-        if (pattern){
-            res = [];
+        if(pattern) {
```

Comment:
Nitpick: Missing space between `if` and `(pattern)`.

Reply:
fixed